CHANGELOG.md

```diff
@@ -2,7 +2,18 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## Unreleased [patch]
+
+_Full changeset and discussions: [#1026](https://github.com/OpenTermsArchive/engine/pull/1026)._
+
+### Added
+
+- Ensure `Reporter` configuration is defined before instantiating it
+- Ensure to add links to `versions` and `snapshots` in issues only if repositories are defined in configuration
+
+### Removed
+
+- No longer define default configuration for `Reporter` module
```

Comment:
I believe this was a breaking change: if the reuser relied on the default configuration and did not define theirs, the behaviour would be significantly altered as they would not have any issues updates anymore.